libs/entry-components/styles/modules/components/checkboxes/_generator.scss

```diff
@@ -0,0 +1,24 @@
+@use 'sass:map';
+
+@mixin generate-from($theme) {
+	$background-color: map.get($theme, 'general', 'checkboxes', 'background');
+	$hover-border-color: map.get($theme, 'general', 'checkboxes', 'border');
+
+	.mat-mdc-checkbox.mat-accent, .mat-mdc-checkbox.mat-primary {
```

Comment:
If we're aware of the fact we wanna override behaviors of checkboxes, why would we put color="primary" or color="accent" on them?